src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1540,13 +1548,98 @@ The LDAP configuration is defined in `PROACTIVE_HOME/config/authentication/ldap.
 
 ==== Set the LDAP server URL
 
-First, you have to define the LDAP's URL of your organization. This address corresponds to the property: `pa.ldap.url`. You have to put a standard LDAP-like URL, for example *ldap://myLdap*. You can also set a URL with secure access: *ldaps://myLdap:636*.
+First, you have to define the LDAP URL of your organization. This address corresponds to the property: `pa.ldap.url`.
+You have to put a standard LDAP-like URL, for example *ldap://myLdap*.
+You can also set a URL with secure access: *ldaps://myLdap:636*.
+
+Finally, it is possible to define additional URLs that will be used as backup if the primary URL is not responding.
+----
+pa.ldap.url=ldap://myLdapServer1 ldap://myLdapServer2
+----
+
+==== Define user and group filters
+
+The authentication module must be able to find a user ldap entry, given its login name (or any other appropriate ldap attribute).
+For each user, the module must also be able to find all group ldap entries associated with the user.
+
+The LDAP module uses ldap filters to search for users and groups.
+
+===== User filters
+
+The user filter is defined using the property `pa.ldap.user.filter`.
+The search is performed according to a configurable ldap subtree, controlled by the property `pa.ldap.userssubtree`.
+
+`pa.ldap.userssubtree` must be defined according to the base distinguished name of the ldap server.
+It can be further refined to restrict the search to a specific subtree where the users are located.
```

Comment:
I deem it is very useful to provide examples of LDAP search commands that allow to check whether the values of the filters are correct.